backend/main_test.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func mustTime(s string) time.Time {
+	t, err := time.Parse("2006-01-02", s)
+	if err != nil {
+		panic(err)
+	}
+	return t
+}
+
+func TestParse(t *testing.T) {
+	s := `
+Date, 	    Coutry, Province, Lat, Long, Confirmed, Recovered, Deaths
+2020-01-01,    Abc,         ,   2,    1,         3,         0,      1
+2020-01-02,    Abc,         ,   2,    1,        10,         2,      6
+2020-01-03,    Abc,         ,   2,    1,        12,         4,      5
+`
+	b := bytes.NewBufferString(s[1:])
+	got, err := parse(b)
+	require.NoError(t, err)
+
+	want := &Data{
+		Dates: []time.Time{mustTime("2020-01-01"), mustTime("2020-01-02"), mustTime("2020-01-03")},
+		Countries: map[string]*Country{
+			"Abc": {
+				Name:      "Abc",
+				Lat:       2,
+				Lng:       1,
+				Confirmed: []int{3, 7, 2},
+				Recovered: []int{0, 2, 2},
+				Dead:      []int{1, 5, -1},
```

Comment:
If this is possible, I think the "country" needs to be Golgotha.

Reply:
or just a country where people do shitty jobs with data entry.

Reply:
so pretty much could be any.